src/main/java/bio/terra/service/filedata/azure/tables/TableDirectoryDao.java

```diff
@@ -84,6 +91,24 @@ public String getPartitionKey(String prefix, String path) {
     return prefix + encodePathAsAzureRowKey(parentDir);
   }
 
+  public void createDirectoryEntries(
```

Comment:
Can I request a docstring on this public method?

Reply:
you bet!